Sources/TemplateKit/Pipeline/TemplateRenderer.swift

```diff
@@ -45,40 +45,27 @@ extension TemplateRenderer {
 }
 
 extension TemplateRenderer {
-    // MARK: Render Path
-    
-    /// Loads and renders a raw template at the supplied path using an empty context.
-    ///
-    /// - parameters:
-    ///     - path: Path to file contianing raw template bytes.
-    /// - returns: `Future` containing the rendered `View`.
-    public func render(_ path: String) -> Future<View> {
-        return render(path, .null)
-    }
-    
-    /// Renders the template bytes into a view using the supplied `Encodable` object as context.
-    ///
-    /// - parameters:
-    ///     - path: Path to file contianing raw template bytes.
-    ///     - context: `Encodable` item that will be encoded to `TemplateData` and used as template context.
-    /// - returns: `Future` containing the rendered `View`.
-    public func render<E>(_ path: String, _ context: E) -> Future<View> where E: Encodable {
+    /// See `ViewRenderer`.
+    public func render<E>(_ path: String, _ context: E, userInfo: [AnyHashable: Any]) -> Future<View> where E: Encodable {
```

Comment:
Does it make sense to add a default value for userInfo of `[:]` here as well?

Reply:
`TemplateRenderer` conforms to `ViewRenderer` which already provides an overload omitting `userInfo`.